app/src/main/res/layout/numberpicker.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (C) 2020 Axel Paetzold
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<NumberPicker xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/yearPicker"
+    android:layout_height="wrap_content"
+    android:layout_width="match_parent"
```

Comment:
Please use `wrap_content` instead and wrap the whole thing into a `LinearLayout`. This makes sure that the number picker looks a bit nicer.

Reply:
Your suggestion looks odd on Android 6, e.g. no centering of the number picker. I will go with the current one apart from `wrap_content`.

Reply:
What exactly you don't like? It looks the same for each Android version.

Reply:
In the Android 6 emulator (API level 23), your suggestion looks the same as mine except that the numbers/years are not centered as in my solution.